paddle/fluid/framework/executor_thread_worker.cc

```diff
@@ -238,6 +240,49 @@ static void print_fetch_var(Scope* scope, const std::string& var_name) {
   VLOG(1) << "print_fetch_var: unrecognized data type:" << tensor.type();
 }
 
+void ExecutorThreadWorker::TrainFilesWithTimer() {
```

Comment:
Should we merge TrainFilesWithTimer with TrainFiles?

Reply:
There seems a lot of duplicated code here. Not sure whether it will affect speed given a lot of conditions. Another way is to use macro which can not be changed during runtime.